lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/persistence/item/ReportUserIdentityItem.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.di.ipv.core.reportuseridentity.persistence.item;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.commons.codec.digest.DigestUtils;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.persistence.item.PersistenceItem;
+
+import java.util.List;
+
+@DynamoDbBean
+@ExcludeFromGeneratedCoverageReport
+@NoArgsConstructor
+@Data
+public class ReportUserIdentityItem implements PersistenceItem {
+    public ReportUserIdentityItem(
+            String userId,
+            String identity,
+            Integer vcCount,
+            List<String> constituentVcs,
+            Boolean migrated) {
+        this.hashUserId = getUserHash(userId);
+        this.userId = userId;
+        this.identity = identity;
+        this.vcCount = vcCount;
+        this.constituentVcs = constituentVcs;
+        this.migrated = migrated;
+    }
+
+    private String hashUserId;
+    @JsonIgnore private String userId;
+    private String identity;
+    private Integer vcCount;
+    private List<String> constituentVcs;
```

Comment:
This can probably just be a comma-separated string - it'll be a lot more compact in the table (and more user-readable as well)

Reply:
Done